datadog_checks_base/datadog_checks/base/log.py

```diff
@@ -53,10 +54,21 @@ def _get_py_loglevel(lvl):
     """
     Map log levels to strings
     """
-    if not lvl:
-        lvl = 'INFO'
-
-    return LOG_LEVEL_MAP.get(lvl.upper(), logging.DEBUG)
+    # In Python2, transform the unicode object into plain string with utf-8 encoding
+    if PY2 and isinstance(lvl, text_type):
+        try:
+            lvl = lvl.encode('utf-8')
+        except UnicodeError:
```

Comment:
If we're going to catch that, we may as well pass `errors="ignore"` to the `encode` call.

Reply:
right, even better at this point would be using `ascii`